themes/classic/templates/checkout/_partials/cart-detailed-totals.tpl

```diff
@@ -36,7 +36,9 @@
               {$subtotal.label}
             {/if}
           </span>
-          <span class="value">{$subtotal.value}</span>
+          <span class="value">
+            {if 'discount' == $subtotal.type}-&nbsp;{/if}{$subtotal.value}
```

Comment:
Not sure if there should be a space between the minus sign and the number. Wdyt @colinegin @LouiseBonnard ?

In the future that should be handled by CLDR

Reply:
I'm in favor of a space between the minus sign and the number.